api/swagger.yml

```diff
@@ -120,6 +120,40 @@ definitions:
         type: string
         enum: [ common_prefix, object ]
 
+  staging_location:
+    type: object
+    description: location for placing an object when staging it
+    properties:
+      physical_address:
+        type: string
+      token:
+        type: string
+        description: opaque staging token to use to link uploaded object
+    required:
+      - token
+
+  staging_metadata:
+    type: object
+    description: information about uploaded object
+    properties:
+      staging:
+        type: object
+        $ref: "#/definitions/staging_location"
+      checksum:
+        type: string
+        description: unique identifier of object content on backing store (typically ETag)
+      mtime:
+        type: integer
+        format: int64
```

Comment:
we probably make this mistake in other places too, but we should at least provide in a description what unit this int represents (i.e. seconds? ms? ns?)

Reply:
So far it has always appeared with this name, not sure whether it is more confusing to introduce a _new_ name (and then users can never figure out the name of the field) or re-use the _old_ (bad) name.
Will change.